tools/dependency/release_dates.py

```diff
@@ -86,12 +86,18 @@ def create_issues(dep, metadata_version, release_date, latest_release):
             labels.append(label.name)
     if len(labels) != len(LABELS):
         raise DependencyUpdateError('Unknown labels (expected %s, got %s)' % (LABELS, labels))
-    body = f'*WARNING* {dep} has a newer release than {metadata_version}@<{release_date}>:{latest_release.tag_name}@<{latest_release.created_at}>'
-    title = f'Newer release available {dep}: {latest_release.tag_name}'
+    body = f'''\
+            Package Name: {dep}
+            Current Version: {metadata_version}@{release_date}
+            Available Version: {latest_release.tag_name}@{latest_release.created_at}
+            Upstream link: https://github.com/{package_repo.full_name}\
+            '''
+    title = f'Newer release available `{dep}`: {latest_release.tag_name} (current: {metadata_version})'
```

Comment:
where the `metadata_version` is a sha hash, i think we can truncate it in the title - perhaps to 7 chars

here is a function that tests whether a string is a hash (hex digest):

```python

def is_sha(text):
    if len(text) != 40:
        return False
    try:
        int(text, 16)
    except ValueError:
        return False
    return True

```

Reply:
ye that can be done!